utility/src/test/java/org/oppia/android/util/parser/image/UrlImageParserTest.kt

```diff
@@ -79,6 +78,82 @@ class UrlImageParserTest {
     assertThat(loadedBitmaps.first()).contains("test_image.svg")
   }
 
+  @Test
+  fun testLoadDrawable_bitmap_fromUrl() {
```

Comment:
Here and elsewhere, suggested rename: `systemUnderTest_arrangement_expectedOutcome`
```suggestion
  fun testLoadDrawable_bitmapFromUrl_loadsBlockImage() {
```

Reply:
Done .